README.md

```diff
@@ -1,28 +1,33 @@
 # 🍜 KNoodle
 
-KNoodle is KNoWS' Solid-based alternative to Doodle.
-It allows you to find time slots that work for different people,
+KNoodle is KNoWS' (and OxfordHCC's) Solid-based alternative to Doodle It allows you to find time slots that work for different people,
 by using their availability calendar which is made available through a Solid pod.
 
-## Run locally via HTTP server
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
 
-1. Install dependencies via `npm i`.
-2. Run production HTTP server via `npm start` (this first runs webpack to generate the browser index.js file), OR
-3. Run development HTTP server (that reloads when you make changes) via `npm run watch`
+## Local development
```

Comment:
No instructions are present on how the install the Node.js dependencies. I know it's only doing `npm i`, but it still should be mentioned.

Reply:
Oops, I'll update the readme